textdb/textdb-perftest/src/main/java/edu/uci/ics/textdb/perftest/excelsink/ExcelSink.java

```diff
@@ -0,0 +1,164 @@
+package edu.uci.ics.textdb.perftest.excelsink;
+
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+import java.io.File;
+import edu.uci.ics.textdb.api.constants.SchemaConstants;
+import edu.uci.ics.textdb.api.dataflow.IOperator;
+import edu.uci.ics.textdb.api.dataflow.ISink;
+import edu.uci.ics.textdb.api.exception.TextDBException;
+import edu.uci.ics.textdb.api.field.IField;
+import edu.uci.ics.textdb.api.schema.Schema;
+import edu.uci.ics.textdb.api.tuple.Tuple;
+import edu.uci.ics.textdb.api.utils.Utils;
+import edu.uci.ics.textdb.perftest.utils.PerfTestUtils;
+
+/**
+ * ExcelSink is a sink that can write a list of tuples into an excel file
+ * setInputOperator -> open -> collectAllTuples -> close -> deleteFile
+ * The path of saved files is "textdb/textdb/textdb-perftest/src/main/resources/index/excel/"
+ * @author Jinggang Diao
+ *
+ */
+public class ExcelSink implements ISink {
+    
+    private IOperator inputOperator;
+    
+    private Schema inputSchema;
+    private Schema outputSchema;
+    
+    private boolean isOpen;
+    
+    private Workbook wb;
+    private FileOutputStream fileOut;
+    Sheet sheet;
+    Row row;
```

Comment:
Make variables `private`

There's no need to maintain a global `row`variable. You can maintain a `cursor`